source/manual/datalabs-start-new-project.html.md

```diff
@@ -0,0 +1,164 @@
+---
+owner_slack: "#govuk-data-labs"
+title: Start a new project
+section: Data Labs Team Manual
+layout: manual_layout
+parent: "/manual.html"
+---
+
+When starting a new data science project, you should:
+
+- store your project in GitHub
+- set up your project using `govcookiecutter`
+- organise your workload using Trello
+- manage your machine's resources with AWS Sagemaker
+
+## Store your projects in GitHub
+
+You must store all your projects in GitHub repositories (repos).
+
+See the [GOV.UK developer documentation on setting up your GitHub account](https://docs.publishing.service.gov.uk/manual/get-started.html#2-set-up-your-github-account) for information on how to get access to GitHub.
+
+You should follow best practice when storing projects in GitHub:
+
+- store GOV.UK projects in [`alphagov` organisation](https://github.com/alphagov) repos
+- store non-GOV.UK projects in [`ukgovdatascience` organisation](https://github.com/ukgovdatascience) repos
+- do not store repos under your GitHub username
+- set your repo to public where possible as the GOV.UK Data Labs team likes to code in the open
+- when naming your repo, use hyphens instead of underscores to separate words.
+- to make sure colleagues have access to repo if the repo owner leaves, grant the following GitHub teams access to your `alphagov` repositories:
+  - admin access: `gov-uk-production`
+  - write access: `gov-uk-data-scientists`
+- for `ukgovdatascience` repos, grant the following GitHub teams access:
+  - write access: `gdsdatascience`
+
+By default, GitHub organisation owners have admin access to all repos. If you cannot access a repo, speak to the owner for access.
+
+See the [GitHub documentation on access permissions](https://docs.github.com/en/get-started/learning-about-github/access-permissions-on-github) for more information.
+
+### Add branch protection rules
+
+You must add branch protection rules to the `main` branch of your repo to prevent unwanted actions to this branch. At a minimum, set that all pull requests require at least one approved review.
+
+1. Go to your GitHub repo.
+1. Select __Settings__ and then select __Branches__ in the left-hand navigation.
+1. In __Branch protection rules__, select __Add rule__.
+1. Enter "main" into the __Branch name pattern__ field.
+1. In __Protect matching branches__, select the __Require pull request reviews before merging__ checkbox. By default this requires one approving review.
+1. Change any other options as necessary.
+1. Select __Save changes__.
+
+## Set up your project using `govcookiecutter`
+
+You should use `govcookiecutter` to set up your data science project.
+
+[`govcookiecutter`](https://github.com/ukgovdatascience/govcookiecutter) is a template for data science projects in HM Government and the wider public sector.
```

Comment:
Here's a [blog post about govcookiecutter](https://dataingovernment.blog.gov.uk/2021/07/20/govcookiecutter-a-template-for-data-science-projects/) that provides an informal introduction.

Reply:
Suggest add this recently published blog link in

Reply:
done